tests/unit/loop-metrics.test.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const test = require('node:test')
+const assert = require('node:assert')
+
+test('Loop Metrics', async (t) => {
+  const MICRO_TO_MILLIS = 1e-3
+  const SPIN_TIME = 2000
+  const CPU_EPSILON = SPIN_TIME * 0.05 // Allowed fudge factor for CPU times in MS
+  const metricEmitter = require('../..')({ timeout: 10 })
+  const testStart = Date.now()
+
+  t.after(() => {
+    metricEmitter.unbind()
+  })
+
+  // Check the structure of the metric object.
+  let metric = metricEmitter.getLoopMetrics().usage
+  assert.strictEqual(typeof metric, 'object', 'should provide a metric object')
+  assert.strictEqual(typeof metric.total, 'number', 'should have a total')
+  assert.strictEqual(typeof metric.min, 'number', 'should have a min')
+  assert.strictEqual(typeof metric.max, 'number', 'should have a max')
+  assert.strictEqual(typeof metric.sumOfSquares, 'number', 'should have a sumOfSquares')
+  assert.strictEqual(typeof metric.count, 'number', 'should have a count')
+
+  // Check that the values are reset after the first call. Since this is
+  // synchronous with the previous call, all results should be zero.
+  metric = metricEmitter.getLoopMetrics().usage
+  assert.deepStrictEqual(
+    metric,
+    {
+      total: 0,
+      min: 0,
+      max: 0,
+      sumOfSquares: 0,
+      count: 0
+    },
+    'should reset all the values'
+  )
+
+  // Queue up a loop with some CPU burn.
+  await new Promise((resolve) => setTimeout(resolve, 100))
+
+  console.log('spinning cpu...')
```

Comment:
Got it. Then I'll completely remove this `console.log` since we can't use it as a comment:

```js
if (process.arch === 'arm64') {
    console.log(
      `{ min: ${usage.min}, max: ${usage.max}, meanTime: ${meanTime}, count: ${usage.count}, total: ${usage.total} }`
    )
  }
  ```